source/meetings/_meeting.erb

```diff
@@ -1,6 +1,6 @@
-<div class="<%= locals[:first] ? 'first ' : '' %>entry" id="<%= meeting.data.title.parameterize %>">
+<article class="<%= locals[:first] ? 'first ' : '' %>entry" id="<%= meeting.data.title.parameterize %>">
   <h2><%= link_to meeting.data.title, meeting %></h2>
```

Comment:
This could be an `<h1>` as well I think.

Reply:
Updated @h-lame :)